packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -688,6 +688,63 @@
     "@consent_analytics_body2": {
         "description": "second paragraph for the consent analytics UI Page"
     },
+    "contact_form_body_android": "OS: Android (SDK Int: {sdkInt} / Release: ${release})\nModel: ${model}\nProduct: ${product}\nDevice: ${device}\nBrand:{brand}",
```

Comment:
I don't think we should translate these. They are meant for us the developers. They don't bring much value of they are for example in Mandarin

Reply:
I understand your point, but it helps users understand what we share with us

Reply:
Okay, but then we still need some explanatory comments so that the translators know what this is all about

Reply:
Descriptions are now added to each placeholder 👍